test/Query.js

```diff
@@ -674,56 +678,50 @@ describe('Query', function (){
   	});
   });
 
-  it('Should allow multiple local indexes and query correctly', function (done) {
+  it('Should allow multiple local indexes and query correctly', async function () {
     var schema = new dynamoose.Schema({
       id: {
         type: String,
         hashKey: true,
-        required: true
       },
       orgId: {
         type: String,
-        index: [{
-          global    : false,
-          name      : 'OrganizationCreateAtIndex',
-          rangeKey  : 'createdAt',
-          throughput: 1
-      }, {
-          global    : false,
-          name      : 'OrganizationExpectedArriveAtIndex',
-          rangeKey  : 'expectedArriveAt',
-          throughput: 1
-      }],
-      required: true,
+        rangeKey: true,
+      },
+      updatedAt: {
+        type: Date,
+        index: {
+          global: false,
+          name  : 'OrganizationUpdatedAtIndex'
+        }
       },
-      expectedArriveAt: Date
+      expectedArriveAt: {
+        type: Date,
+        index: {
+          global: false,
+          name  : 'OrganizationExpectedArriveAtIndex'
+        }
+      }
     },{
       throughput: 1,
       timestamps: true
     });
-    var Log = dynamoose.model('Log-1', schema);
+    var Log = dynamoose.model('Log-2', schema);
 
     var log1 = new Log({id: "test1", orgId: "org1", expectedArriveAt: Date.now()});
-    log1.save(function() {
-      Log.query('orgId').eq("org1")
+    var log2 = new Log({id: "test1", orgId: "org2", expectedArriveAt: Date.now()});
+
+    await log1.save();
+    await log2.save();
+
+    var res = await Log.query('id').eq("test1")
```

Comment:
Same thing here. Before it was `.query('orgId').eq("org1")`. I'm considered that this isn't testing #343 anymore.